app/main.py

```diff
@@ -2,0 +1,14 @@
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] != "cp":
+        print("Invalid command format.")
+        return
+    source_file, destination_file = command_parts[1], command_parts[2]
```

Comment:
Code Style: It's better to use more descriptive variable names. Instead of `source_file` and `destination_file`, you could use `source_filename` and `destination_filename` to make it clear that these variables contain the names of the files, not the files themselves.